package/MDAnalysis/analysis/hole2/helper.py

```diff
@@ -0,0 +1,556 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
I think util is probably what we’d call this submodule.

Reply:
Changed name to ``utils`` to match ``encore.utils``